### ISSUE TITLE
How can switch from electron-packager to electron-builder?

### ISSUE THREAD

**Issue by twelve6** [SimulatedGREG/electron-vue#1012]
Hello!

What would be the best way to accomplish switching from electron-packager to electron builder?

I have built another instance and migrated my code over, but I think there's gotta be a better way so I can just do it in the current repo instead of having to migrate.

can I just install electron-builder and transfer over the package.json scripts and build settings?

Can both electron-packager and electron-builder live in the same environment? 

Basically what modification would need to be made, if at all possible?

I couldn't find anything on this and now that we built our app and have more knowledge of packager vs builder we would definitely like to go with electron-builder instead. Appreciate and help, thank you!
